torchrl/modules/distributions/discrete.py

```diff
@@ -66,7 +64,7 @@ class OneHotCategorical(D.Categorical):
             reparameterized samples.
             ``ReparamGradientStrategy.PassThrough`` will compute the sample gradients
              by using the softmax valued log-probability as a proxy to the
-             samples gradients.
+             sample gradients.
```

Comment:
Note: I _think_ the `num_params` class attribute defined at l. 82/84 is not used. If that's correct, I can take the opportunity to remove this in this MR.